### PR TITLE
RELATED: BB-1411 subtotals submenu documentation

### PR DESCRIPTION
**Pull request by yanlipnican**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/73/ : SUCCESS in 37s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/55/ : SUCCESS in 8s

File: docs/pivot_table_component.md [hunk 1]
```diff
@@ -200,15 +200,16 @@ To avoid this gap, specify the maximum height of the table using the `maxHeight`
 ## Configuration menu
 
 You can configure the following settings:
+* Totals and subtotals. If you enable subtotals menu but disable totals, subtotals will be disabled too.
```
Comment (evgeniabogranskaya) [inline docs/pivot_table_component.md]: * Totals and subtotals. If you enable the subtotals menu but disable totals, subtotals will be disabled too.

**Comment by evgeniabogranskaya** [comment]
One comment to resolve, otherwise LGTM

File: docs/pivot_table_component.md (hunk 1 above)
Comment (yanlipnican) [reply docs/pivot_table_component.md]: done

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/74/ : SUCCESS in 57s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/56/ : SUCCESS in 11s

**Comment by evgeniabogranskaya** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/75/ : SUCCESS in 31s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/22/ : SUCCESS in 54s

**Pull request merged**